pkgdown-multisite.R

```diff
@@ -21,6 +21,7 @@ html_template = function(fname = ""){
 
 args = commandArgs(trailingOnly=TRUE)
 if(length(args) != 2){
+  cat(paste("arg: ", args, "\n"))
```

Comment:
```suggestion
  cat(paste("arg: ", paste(args, collapse = " "), "\n"))
```
This will return 
`arg: arg1 arg2 arg3`
instead of 
```
arg: arg1 
arg: arg2 
arg: arg3
```

Reply:
both forms are similar, but in in current we can detect also space in parameter

Reply:
ok